### ISSUE TITLE
geotag KML altitude dropping

### ISSUE THREAD

**Issue by jidanni** [exiftool/exiftool#176]
geotag.html says
> the following tags are written to the image (if the corresponding information is available):
> ...GPSLatitude        GPSLongitude      GPSAltitude...

`exiftool -geotag`, yes, does write the first two,
but not GPSAltitude.
```
exiftool *_original|grep -i altitude.*[0-9]|sort
GPS Altitude                    : 9229.7 m Above Sea Level
GPS Altitude                    : 9230.4 m Above Sea Level
exiftool *jpg      |grep -i altitude.*[0-9]|sort
```
(It might as well leave the old altitudes in if it doesn't have any to insert.)

But why might it not have any to insert?
But certainly my log file altitudes'
`<coordinates>-177.028370,51.529266,9754.00</coordinates>`
pass Geotag.pm's
```
undef $$fix{alt} if defined $$fix{alt} and $$fix{alt} !~ /^[+-]?\d+\.?\d*/;
```

All I know is
`gpsbabel -i kml -f FlightAware_*_KSFO_RCTP_*.kml -x interpolate,time=1 -o kml,units=a,extrude=1 ...`)
created  [great log files](#173), that even has `<description>`s like
<table>
<tr><td>Longitude: -177.028370 </td></tr>
<tr><td>Latitude: 51.529266 </td></tr>
<tr><td>Altitude: 32001.312 ft </td></tr>
<tr><td>Speed: 551.5 knts </td></tr>
<tr><td>Heading: 286.0 </td></tr>
<tr><td>Time: 2022-12-21T03:23:56Z </td></tr>
</table>
in it (not that I expect geotag.pm to read '<description>'s),
but at least it read the '<coordinates>' but then never did anything
with the third parameter.

**Comment by boardhead** [comment]
Thanks.  This will be fixed in 12.55

**Comment by boardhead** [comment]
Version 12.55 is now available
